libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.h

```diff
@@ -128,17 +128,6 @@ typedef void (^SFOAuthBrowserFlowCallbackBlock)(BOOL);
  */
 - (void)oauthCoordinatorDidAuthenticate:(SFOAuthCoordinator *)coordinator authInfo:(SFOAuthInfo *)info;
 
-/**
- Sent if authentication fails due to an error. Note: This method is deprecated.  You should use the
- `oauthCoordinator:didFailWithError:authInfo` method instead.
- 
- @param coordinator The SFOAuthCoordinator instance processing this message
- @param error       The error message
- 
- @see SFOAuthCoordinator
- */
-- (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator didFailWithError:(NSError *)error __attribute__((deprecated));
```

Comment:
Shouldn't we wait for `Mobile SDK 9.0` to remove this, to avoid breaking semver?

Reply:
@bhariharan This has been deprecated several versions ago. It should have been removed in 8.0.

Reply:
Sounds good.